apps/src/p5lab/spritelab/commands/criterionCommands.js

```diff
@@ -49,11 +49,14 @@ export const commands = {
   },
 
   // Return true if any sprite's speech include values from a given object.
-  anySpeechIncludesValues(object) {
+  anySpeechIncludesValues(object, object2) {
```

Comment:
what are `object` and `object2`? If it's `currentVar` and ``previousVar` we should probably call them that so it's clear.

Reply:
I think this piece in your PR description was really helpful context:

> in case the author would like to give the flexibility of printing the value from either the current or previous frame

Could be good to summarize in the comment as well!

Reply:
I thought about this and wasn't sure what made the most sense. Labels like `currentVar` reflect the intended use, but technically the function could be used with any object(s).